src/server/modules/user/services/UrlManagementService.ts

```diff
@@ -60,9 +63,9 @@ export class UrlManagementService implements interfaces.UrlManagementService {
           mimetype: file.mimetype,
         }
       : undefined
+
     // Success
-    dogstatsd.increment('shortlink.create', 1, 1, [`isfile:${!!file}`])
-    return this.urlRepository.create(
+    const result = await this.urlRepository.create(
```

Comment:
just rebased on `develop` to resolve some merge conflicts. reordered this bit to put `dogstatsd.increment` only after a URL was successfully created, hope it's okay!